normalizer/tests/test_jageocoder.py

```
@@ -0,0 +1,220 @@
+    
+    for case in test_addresses:
+        print(f"\nテスト: {case['description']}")
+        print(f"住所: {case['address']}")
+        
+        # Jageocoder APIで座標取得
+        lat, lon = get_jageocoder_latlng(case['address'], case.get('area'))
+        
+        if lat is not None and lon is not None:
+            print(f"[OK] Jageocoder座標: {lat:.6f}, {lon:.6f}")
+        else:
+            print(f"[NG] Jageocoderで座標を取得できませんでした")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)